Create ui-c8y-1020-0-16-when-the-modal-for-maximum-number-of-archives-is-cancelled-by.md (#2158)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-0-16-when-the-modal-for-maximum-number-of-archives-is-cancelled-by.md

---
date: ""

> title: Error message no longer shown when user cancels maximum number of archives modal

Reviewer: ```suggestion
title: Canceling the dialog on reaching the maximum number of archives no longer displays an error message
```

> title: When the modal for maximum number of archives is cancelled by the user, an error message is no longer displayed. [GRAFT][release/cd] (#6498)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Error message no longer shown when user cancels maximum number of archives modal
```

product_area: Application enablement & solutions
change_type:
  - value: change-VSkj2iV9m
    label: Fix
component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58926
version: 1020.0.16
---

> When the modal for maximum number of archives is cancelled by the user, an error message is no longer displayed. [GRAFT][release/cd] (#6498)

Author: The Claude AI suggested to use the following body:
```suggestion
In the past, when a user reached the maximum number of archives and the corresponding modal dialog was shown, canceling the modal would incorrectly display an error message. This has been fixed. Now, when a user cancels the maximum number of archives modal, no error message is shown anymore and the application behaves as expected. This change improves the user experience by avoiding unnecessary and confusing error messages.
```

> In the past, when a user reached the maximum number of archives and the corresponding modal dialog was shown, canceling the modal would incorrectly display an error message. This has been fixed. Now, when a user cancels the maximum number of archives modal, no error message is shown anymore and the application behaves as expected. This change improves the user experience by avoiding unnecessary and confusing error messages.

Reviewer: ```suggestion
In the past, when a user reached the maximum number of archives and the corresponding dialog was shown, canceling the dialog would incorrectly display an error message. This has been fixed. Now, when a user cancels the maximum number of archives dialog, no error message is shown any longer and the application behaves as expected. 
```
